plugins/org.jkiss.dbeaver.ext.mssql.ui/plugin.xml

```diff
@@ -56,6 +56,7 @@
             <objectType name="org.jkiss.dbeaver.ext.mssql.model.SQLServerDatabaseTrigger"/>
             <objectType name="org.jkiss.dbeaver.ext.mssql.model.SQLServerDatabaseTrigger"/>
```

Comment:
Hmm... String 57 looks precisely like string 56)

Reply:
Not related to my PR to be quite honest.